internal/config/config_test.go

```diff
@@ -85,7 +87,7 @@ func TestNewConfig_MissingConfigWithDefault(t *testing.T) {
 		ResultPath:       "tmp/rspec.json",
 	}
 
-	if diff := cmp.Diff(c, want); diff != "" {
+	if diff := cmp.Diff(c, want, cmpopts.IgnoreUnexported(Config{})); diff != "" {
```

Comment:
`errs` is unexported field of the config struct. IgnoreUnexported option ignores the unexported fields during comparison.